docs/bosh/README.md

```diff
@@ -398,13 +451,16 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Download [ca_cert.pem](../ca_cert.pem) from this repository.
```

Comment:
The certificate is now pasted in like the director manifest. Using `bosh2 int` would require copying the generated `certs.yml` that correspond to the hardcoded certificates in the director manifest along with the `bosh2` invocation.

I've included a note which provides the ability to delve deeper into certificate handling.